pom.data.xml

```diff
@@ -0,0 +1,550 @@
+        <property>
+          <name>!java-lts</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>${maven-compiler-plugin.version}</version>
+            <configuration>
+              <source>1.8</source>
+              <target>1.8</target>
+              <showWarnings>true</showWarnings>
+              <showDeprecation>true</showDeprecation>
+              <compilerArgument>-Xlint:all</compilerArgument>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>java-lts</id>
+      <activation>
+        <property>
+          <name>java-lts</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>${maven-compiler-plugin.version}</version>
+            <configuration>
+              <source>11</source>
+              <target>11</target>
+              <showWarnings>true</showWarnings>
+              <showDeprecation>true</showDeprecation>
+              <compilerArgument>-Xlint:all</compilerArgument>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- Customise the javadoc plugin to not include module directories in links. We don't
+         currently use modules, so search results were being directed to an 'undefined'
+         directory (for an 'undefined' module). This no-module-directories flag fixes
+         this, but it is only required for JDK 9+ -->
+    <profile>
+      <id>javadoc-no-modules-fix</id>
+      <activation>
+        <jdk>[9,)</jdk>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <configuration>
+              <additionalJOption>--no-module-directories</additionalJOption>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>non-shippping-modules</id>
+      <activation>
+        <property>
+          <name>include-non-shipping-modules</name>
+        </property>
+      </activation>
+      <modules>
+        <module>./eng/spotbugs-aggregate-report</module>
+        <module>./eng/code-quality-reports</module>
+      </modules>
+    </profile>
+
+    <profile>
+      <id>template-module</id>
+      <activation>
+        <property>
+          <name>include-template</name>
+        </property>
+      </activation>
+      <modules>
+        <module>./sdk/template/azure-sdk-template</module>
+      </modules>
+    </profile>
+  </profiles>
+
+  <modules>
+    <module>./batch/data-plane</module>
```

Comment:
Shouldn't these be updated to the new parent azure-data-sdk-parent?

Reply:
Forgot that. Thanks for catching it!